test/forcing/CsvPerFeatureForcingProvider_Test.cpp

```diff
@@ -164,3 +176,63 @@ TEST_F(CsvPerFeatureForcingProviderTest, TestGetAvailableForcingOutputs)
 
 }
 
+///Test CSV Units Parsing
```

Comment:
Might be good to add a test for extra characters after the unit brackets, e.g. `var (unit) extra, ...`